test/driver.js

```diff
@@ -431,6 +431,9 @@ class Driver {
       task.stats = { times: [] };
       task.enableXfa = task.enableXfa === true;
 
+      // just in case scale is not set, setting to default=1
+      task.scale = task.scale || 1;
```

Comment:
I believe that `task.scale ||= 1;` ought to work, given the browsers that we run tests in.